src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureAreaCalculable, FigureDraw {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition.